playground/playground.js

```diff
@@ -30,6 +30,10 @@ window.onload = function() {
         document.location = '#' + document.getElementById('projectId').value;
         location.reload();
     };
+    document.getElementById('createEmptyProject').onclick = function () {
```

Comment:
Looks like `addEventListener` is [generally](https://github.com/TheBrokenRail/scratch-vm/blob/736cfb9145d9a959930e99d7d706a0abfe474ba4/playground/playground.js#L165) [used](https://github.com/TheBrokenRail/scratch-vm/blob/736cfb9145d9a959930e99d7d706a0abfe474ba4/playground/playground.js#L183) [everywhere](https://github.com/TheBrokenRail/scratch-vm/blob/736cfb9145d9a959930e99d7d706a0abfe474ba4/playground/playground.js#L194) [else](https://github.com/TheBrokenRail/scratch-vm/blob/736cfb9145d9a959930e99d7d706a0abfe474ba4/playground/playground.js#L194) (rather than directly setting the `on<event>` method).. maybe it should be used here too?